packages/server/lib/plugins/dev-server.js

```diff
@@ -16,6 +16,7 @@ plugins.registerHandler((ipc) => {
   })
 
   ipc.on('dev-server:compile:success', () => {
+    // eslint-disable-next-line no-undef
```

Comment:
This actually seems like a mistake that's being caught by `no-undef`. If it's not defined, this will set a global variable, which I can't imagine is the intention. The comma at the end of the line is odd too.

@dmtrKovalenko, it looks like you originally wrote this. Any thoughts?

Reply:
`allowToRequest` does not appear anywhere (else) in the entire code-base. Seems like either a mistake or old code.